recipes/whatshap/meta.yaml

```diff
@@ -1,13 +1,15 @@
 package:
   name: whatshap
-  version: "2.2"
+  version: "2.3"
 
 source:
-  url: https://files.pythonhosted.org/packages/00/f3/2e2c850755629a151c2027b636f253745bbbfeee052870f03e99f94a4996/whatshap-2.2.tar.gz
-  sha256: 4cd34e9b82930c4f42e9e6b7dce2e321e4c81f934fdb980b6093ad91a06ae30a
+  url: https://files.pythonhosted.org/packages/37/99/7f156642064fe65aabbe6fa044a485e2aae6437dd545a9d539c06c1535dd/whatshap-2.3.tar.gz
+  sha256: 1f7ac47b18784b213492e4b92dddc63132d371b2f353660462536bcadc15e62e
 
 build:
-  number: 1
+  number: 0
+  # Unexplicable CI failure, feel free to try to make this work again
+  skip: true  # [osx]
```

Comment:
The CI log seems to be terribly mangled... I hope we can get that fixed in the future.
What failed was
```
2024-05-05T14:15:16.7690310Z 14:15:10 [32mBIOCONDA INFO[0m (ERR) subprocess.CalledProcessError: Command '['/opt/mambaforge/envs/bioconda/conda-bld/whatshap_1714917823295/_build_env/bin/llvm-otool', '-l', '/opt/mambaforge/envs/bioconda/conda-bld/whatshap_1714917823295/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.9/site-packages/whatshap/polyphase/solver.cpython-39-darwin.so']' died with <Signals.SIGABRT: 6>.[0m
```
With LLVM 15.0.7 ; I agree, we can skip macOS for now, and just try to rebuild this soon with updated compilers etc.